apps/agreement/test/agreement/agreement_lock_manager.js

```diff
@@ -0,0 +1,82 @@
+const deployer = require('../helpers/utils/deployer')(web3, artifacts)
+const { injectWeb3, injectArtifacts } = require('@aragon/contract-helpers-test')
```

Comment:
Not opposed to leaving this here, but since we don't seem to use any part of the `contract-helpers-test` module, we could remove it.